README.md

```diff
@@ -1,4 +1,4 @@
-The CyberArk Conjur Buildpack is a [decorator buildpack](https://github.com/cf-platform-eng/meta-buildpack#what-is-a-decorator) that provides convenient and secure access to secrets stored in Conjur.
+The CyberArk Conjur Buildpack is a [supply buildpack](https://docs.cloudfoundry.org/buildpacks/custom.html#contract) that provides convenient and secure access to secrets stored in Conjur.
 
 The buildpack carries out the following:
```

Comment:
I think more accurately the buildpack installs some scripts that do the following